app/views/admin/admin_users/confirm_delete_user_creations.html.erb

```diff
@@ -0,0 +1,32 @@
+
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts("Delete Spammer Creations") %></h2>
+
+<!--main content-->
+<%= form_tag destroy_user_creations_admin_user_path(@user), method: :post, class: "simple destroy" do %>
+
+  <p class="caution notice">
+    <%= ts('Are you sure you want to <strong><em>delete</em></strong> all the works and comments created by this user, along with their %{bookmarks} bookmarks, %{series} series, and %{collections} collections? This <strong>cannot be undone</strong>.', bookmarks: @bookmarks.count, series: @series.count, collections: @collections.count, comments: @comments.count).html_safe %>  
```

Comment:
I see `comments: @comments.count`,  but it looks like the number of comments isn't actually displayed in the message, so I think it can be removed.